cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -55,10 +55,21 @@ CREATE TABLE IF NOT EXISTS epa_violations(
     violation_code varchar(255) NOT NULL,
     compliance_status varchar(255) NOT NULL,
     start_date DATE NOT NULL,
+    end_date DATE NOT NULL,
     pws_id varchar(255) NOT NULL REFERENCES water_systems(pws_id),
     PRIMARY KEY(violation_id)
     );
 
+-- Violation counts --
+CREATE VIEW violation_counts AS
+SELECT
+    pws_id,
+    geom,
+    COUNT(violation_id)
```

Comment:
I think it'd be more useful to show the violation rate (or service line, or per capita) on the frontend, rather than the raw count. What's the best way to support that here? Should this view do that calculation?

Reply:
Delaying this to another PR at this point but will do!